test/tests_granularity.jl

```diff
@@ -0,0 +1,9 @@
+@testset "granularity" begin
+        timestamps = collect(DateTime(2000, 2, 27):Hour(1):DateTime(2000, 10, 5))
+        vals = ones(length(timestamps))
+        ts = TimeSeries("teste", timestamps, vals)
+        ts = hourly_to_monthly(ts)
+        @test ts.name == "teste"
+        @test all(ts.timestamps .== collect(DateTime(2000, 2, 1):Month(1):DateTime(2000, 10, 1)))
+        @test all(ts.vals .== 1)
```

Comment:
you are not testing every function... And you can add more elaborate tests, something like no observations in a month and some weird cases